src/core/dappletEngine.ts

```diff
@@ -23,26 +19,8 @@ export class DappletEngine {
 
     // }
 
-    public async load(): Promise<void> {
-        const promises = this._frameExecutors.map(f => f.load(this._context.dappletProviders));
-        await Promise.all(promises);
-    }
-
-    public validate() {
-        this._frameExecutors.map(f => f.validate());
-    }
-
     async run(): Promise<void> {
         const promises = this._frameExecutors.map(f => f.run());
         await Promise.all(promises);
```

Comment:
do you need `return` here?

Reply:
I'm writing this part and don't know yet...